aten/src/ATen/native/Normalization.cpp

```diff
@@ -466,6 +468,15 @@ Tensor layer_norm(const Tensor& input, IntArrayRef normalized_shape,
     const Tensor& weight /* optional */, const Tensor& bias /* optional */,
     double eps, bool cudnn_enabled) {
 
+    if (!input.is_cuda()) {
```

Comment:
hmm, I guess we don't have `is_cpu` :(.  I'd make this `input.device().type() == kCPU` or similar since we have other devices now.

Reply:
input.device().is_cpu()

Reply:
Done